### PR TITLE
[ECMSCC-9663] CAM CLC Management problem

### PR DESCRIPTION
**Pull request by yongjiel**

### CODE REVIEW

**Comment by yongjiel** [comment]
There is a json bug in APIv2/server.py line 356.  Fixed with 'json.dump([self.id])' instead of '["%s"]' %s self.id.  Now it works.
Test  result:
CLC provider. 2 windows instances, 2 centos instances:
1. deploy  OK.
2. shutdown. OK
3. poweron. OK.
4. terminate. OK.

File: src/clc/APIv2/server.py [hunk 1]
```diff
@@ -67,9 +67,8 @@
 
 
 class Servers(object):
-
-	def __init__(self,servers_lst,alias=None,session=None):
-		"""Container class for one or more servers.
+    def __init__(self, servers_lst, alias=None, session=None):
+        """Container class for one or more servers.
```
Comment (jinzishuai) [inline src/clc/APIv2/server.py:71]: why so many formatting change?

Comment (jinzishuai) [reply src/clc/APIv2/server.py:71]: if you think it is a 1-line fix, please submit a one-line fix PR. This one is mixed with lots of format change. Even if we want to fix the format, we need a separate PR just for that.

**Comment by Ahhmyface** [comment]
remove formatting changes. otherwise approved

File: src/clc/APIv2/server.py (hunk 1 above)
Comment (yongjiel) [reply src/clc/APIv2/server.py]: It is python black in my VScode.

**Pull request merged**